WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleUnsupportedWidthView.swift

```diff
@@ -10,7 +10,7 @@ struct PointOfSaleUnsupportedWidthView: View {
             VStack(alignment: .center, spacing: PointOfSaleItemListErrorLayout.headerSpacing) {
                 Image(systemName: "ipad.landscape.badge.exclamationmark")
                     .symbolRenderingMode(.palette)
-                    .foregroundStyle(Color(.wooCommerceAmber(.shade60)), Color.posPrimaryText)
+                    .foregroundStyle(Color.posAlert, Color.posAlert)
```

Comment:
I opted to use the same colour here for consistency: 

<img width="911" alt="20250212_width_not_supported" src="https://github.com/user-attachments/assets/77dc1775-d573-4b0a-af90-6216340939a1" />

Reply:
Actually there are changes on a [newest PR](https://github.com/woocommerce/woocommerce-ios/pull/15119/files) that uses `.onSurface` and looks better, I'll adapt this one before merging:

![simulator_screenshot_D9766B86-6F85-42C8-9544-82638CFB6C23](https://github.com/user-attachments/assets/04ec8279-c224-4945-9d5e-a793d0ba34b9)